lib/src/base-styles/_nx-tile.scss

```diff
@@ -136,10 +136,14 @@ $nx-tile-background-color: $nx-white;
 
 // On pages with tiles with multiple sections these are lighter weight titles
 .nx-tile__section-header {
-  border-top: 1px solid #b7bfdb;
-  margin-top: $nx-spacing-l;
+  margin-top: $nx-spacing-xl;
   margin-bottom: $nx-spacing-md;
-  padding-top: $nx-spacing-l;
+
+  &:first-of-type {
```

Comment:
This approach presents a problem in conjunction with any interleaving elements, such as the `section.gallery-example-code` elements that `GalleryExampleTile` produces.  See the `NxFontAwesomeIcon` page for an example a a multi-section `GalleryExampleTile`

Reply:
> This approach presents a problem in conjunction with any interleaving elements, such as the `section.gallery-example-code` elements that `GalleryExampleTile` produces. See the `NxFontAwesomeIcon` page for an example a a multi-section `GalleryExampleTile`

After discussion I think this works well. https://github.com/sonatype/sonatype-react-shared-components/pull/64/commits/6a793c18e18b08d1a354e5b78125af7b73c1c654

Is it worth refactoring the other `nx-tile` examples to use the more semantic `<section>`, `<header>`, and `<footer>` tags?

Reply:
> Is it worth refactoring the other nx-tile examples to use the more semantic <section>, <header>, and <footer> tags?

Maybe the ones on nx-tile documentation page, sure.  I don't know that we need to refactor the whole gallery though.  I'd consider the use of those semantic elements to be optional but recommended — not required.